src/components/queryBuilder/views/TraceQueryBuilder.tsx

```diff
@@ -102,6 +124,18 @@ export const TraceQueryBuilder = (props: TraceQueryBuilderProps) => {
     }));
   }, builderState);
 
+  // A function to assemble custom column definitions for the prefixed Events/Links columns
+  const onColumnPrefixChange = (name: keyof TraceQueryBuilderState, columnHint: ColumnHint) => {
```

Comment:
This change is necessary since at the moment, you cannot type into the "Events Prefix" field in the column selection form.

This function generates a `SelectedColumn` struct for the arbitrary input, which is necessary since when `flatten_nested=1`, the `ColumnSelect` component recognises the `Events` column as multiple sub-columns.